### DISCUSSION TITLE
Segoe Fluent Icons in FluentAvalonia.UI.Controls.ItemElement

### DISCUSSION THREAD

**Discussion by jasin** [amwx/FluentAvalonia#544] [Q&A]
I was looking for some icons to use for my app. I eventually stumbled upon Symbol.cs which is just an enum. 
I noticed that file is incomplete according to [Segoe Fluent Icons Font](https://learn.microsoft.com/en-us/windows/apps/design/style/segoe-fluent-icons-font#pua-ea00-ec00) at microsoft website. 

I also noticed the Symbol.cs said not to manually edit this file(I assume because it will be overriden by updates). There seems to be quite a few missing icons that are available if you just use the decimal equivalent of the hex shown on the above microsoft website. IE. Bank 0xE825 or 59429 the decimal equivalent.

Is there a reason for this? Pull request

**Comment by robloo** [comment]
The enum in Symbol.cs is based on the one in WinUI (https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.symbol?view=winrt-22621) with some FluentAvalonia specific additions. It is ONLY for the primary-used icons and is intended for use with SymbolIcon (https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.symbolicon?view=winrt-22621).

In FluentAvalonia, Symbol.cs was originally generated using the tool here: https://github.com/robloo/SymbolIconManager/blob/main/Source/Models/Specialized/FluentAvalonia.cs. Although it may have since been forked.

I don't think it within scope to modify the Symbol.cs file (as it's mostly to match WinUI). However, if the underlying fluent avalonia font is missing an icon you need it can be added and the font re-built. If you need an icon not in Symbol.cs just use the unicode point and fluent avalonia font with FontIcon.
